src/Platforms/AbstractPlatform.php

```diff
@@ -1533,17 +1533,7 @@ public function getUniqueConstraintDeclarationSQL(UniqueConstraint $constraint):
             $chunks[] = 'CLUSTERED';
         }
 
-        $columnSQL = [];
-
-        foreach ($columns as $column => $definition) {
-            if (is_array($definition)) {
-                $columnSQL[] = $column;
-            } else {
-                $columnSQL[] = $definition;
-            }
-        }
-
-        $chunks[] = sprintf('(%s)', implode(', ', $columnSQL));
+        $chunks[] = sprintf('(%s)', implode(', ', $columns));
```

Comment:
If `$columns` is always a list of strings, the whole `foreach` loop doesn't make sense. 🧐